### PR TITLE
P0 explore

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Clean understandable code
Good work prabhat
Good to merge

File: src/pages/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.other-option-container{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.show-hide-icon{
+    font-size: 1.5rem;
+    position: absolute;
+    bottom: 12px;
```
Comment (devAmlan) [inline src/pages/Login/login.css:20]: you can use rem here

**Review by devAmlan — APPROVED** [review]
Clean UI, code looks good,
Suggested a small change please check.

File: src/index.js [hunk 2]
```diff
@@ -11,9 +11,11 @@ makeServer();
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
+    <AuthProvider>
     <VideoProvider>
         <App />
     </VideoProvider>
+    </AuthProvider>
```
Comment (bossmonkey98) [inline src/index.js:18]: you use a wrapper for all the providers

**Comment by bossmonkey98** [comment]
img
When I visit the explore page the size of cards are different you need to resize the thumbnail images,
Remove borders from all the Cards, Add borders only on hover also use transform scale and transition to enhance UI experience
Try not to use Italic text its not that eye catching
In the Home page there is lot of space between categories cards

File: src/components/VideoCard/VideoCard.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        <>
+          <div className="video-card margin-xs padding-xs">
+              <div className="img-container">
+                <img className="img-responsive" src={thumbnail} alt="video_thumbnail"/>
+              </div>
+              <div>
+                  <h4 className="video-title text-sm">{title}</h4>
+                  <p className="video-creator text-md">{creator}</p>
+              </div>
+          </div>
```
Comment (bossmonkey98) [inline src/components/VideoCard/VideoCard.js]: use avatars along with creator name
keep the font size of title greater then creater

**Review by bossmonkey98 — APPROVED** [review]

**Pull request merged**